index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <h1 id="title">Survey Form</h1>
+  <p id="description">This is our Form</p>
+  <form action="post" id="survey-form">
+    <label for="name" id="name-label">Name</label>
+    <br>
+
+    <input type="text" id="name" placeholder="Enter name" required>
+
+    <br>
+    <label for="email" id="email-label">Email</label>
+    <br>
+
+    <input type="email" id="email" placeholder="Enter email" required>
+    <br>
+    <label for="name" id="number-label">Number</label>
+    <br>
+
+    <input type="number" id="number" placeholder="Enter number" min="5" max="10">
+    <br>
+    <p>What is your favourit animal</p>
+    <select id="dropdown">
+      <option value="cat">cat</option>
+      <option value="dog">dog</option>
+    </select>
+    <br>
+    <p>Would you get an animal to live with you?</p>
+
+    <input type="radio" id="choice1" name="animals" value="sure">
+    <label for="choice1">Sure</label>
+
+    <input type="radio" id="choice2" name="animals" value="Never">
+    <label for="contactChoice2">Never</label>
+
+
```

Comment:
- [ ] [OPTIONAL] Nice job being polite, and giving reasons for the requested change :+1:, however I suggest you also phrase your comments as suggestions instead of orders

Example:
:x: Wrong
Kindly remove double spacing, it will make your code clean and readable.

:heavy_check_mark: Right
I Kindly suggest that you remove double spacing, it will make your code clean and readable.